search_gov_crawler/domains/config/domains_elasticsearch.libsonnet

```
@@ -3316,7 +3316,7 @@ local output_target = 'elasticsearch';
   {
     name: 'tricare_nhcc',
     config: DomainConfig(allowed_domains='charleston.tricare.mil',
-                         starting_urls='http,://harleston.tricare.mil/',
+                         starting_urls='http://charleston.tricare.mil/',
```

I think we should change this to httpS:
```suggestion
                         starting_urls='https://charleston.tricare.mil/',
```

---

good catch!